app/components/public/add-to-calender.hbs

```diff
@@ -9,15 +9,41 @@
       {{/if}}
       {{general-date @event.startsAt 'tz' tz=@event.timezone}}
     </div>
-    <UiDropdown class="ui floating dropdown button mt-4">
-      {{t 'Add to Calendar'}}
-      <div class="menu">
-        {{#each this.calendarUrls as |calendar|}}
-          <a href={{calendar.url}} class="item" target="_blank" rel="noreferrer nofollow">
-            {{calendar.name}}
-          </a>
-        {{/each}}
+
+    <div class="ui grid mt-4">
+      <div class="eight wide column p-1">
+        <UiDropdown class="ui floating fluid dropdown button mb-2">
+          {{#if @event.sessions}}
```

Comment:
read previous review

Reply:
I didn't found any other way than that to make button equals.

Reply:
https://github.com/fossasia/open-event-frontend/pull/6088#discussion_r547534803